### PR TITLE
[WIP] Update for Preact 10

### PR DESCRIPTION
**Pull request by developit**

### CODE REVIEW

File: test/render.js [hunk 1]
```diff
@@ -206,7 +206,7 @@ describe('render', () => {
 					match({
 						foo: 1,
 						children: [
-							match({ nodeName: 'span', children: ['asdf'] })
+							match({ nodeName: 'span', children: 'asdf' })
```
Comment (sync) [inline test/render.js]:
```suggestion
							children: match({ type: 'span', props: { children: 'asdf' } })
```

File: test/render.js [hunk 2]
```diff
@@ -284,9 +284,7 @@ describe('render', () => {
 				.and.calledWithExactly(
 					match({
 						foo: 1,
-						children: [
-							match({ nodeName: 'span', children: ['asdf'] })
-						]
+						children: match({ nodeName: 'span', children: ['asdf'] })
```
Comment (sync) [inline test/render.js]:
```suggestion
						children: match({ type: 'span', props: { children: 'asdf' } })
```

**Comment by sync** [comment]
Have a look at this PR, had a go at fixing all failing tests: 
https://github.com/developit/preact-render-to-string/pull/71

**Comment by developit** [comment]
Thanks a bunch @sync!

**Pull request merged**